hack/verify-e2e.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# Copyright 2014 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Verify that E2E's use Describe wrappers, so that we can auto tag and provide
+# other wrapper functionality for the entire suite.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
+
+cd "${KUBE_ROOT}"
+
+find_files() {
+  find ./test/e2e/ -name '*.go'
+}
+
+# Any file declaring a Describe implementation that doesn't call the wrapper
+# is at risk of not following conventions we've implemented.
+kdescribe_bad_files=$(find_files | xargs grep "Describe(" | grep -v "KubeDescribe(" | grep -v "skipe2everify")
+if [[ -n "${kdescribe_bad_files}" ]]; then
+  echo "The following files appear to be violating conventions for how we use ginkgo."
```

Comment:
Can you make this say something like "The following files use Describe instead of KubeDescribe"?